modules/planning/tasks/deciders/path_reference_decider/path_reference_decider.cc

```diff
@@ -61,8 +64,20 @@ Status PathReferenceDecider::Process(Frame *frame,
     reference_line_info->mutable_path_data()->set_is_valid_path_reference(
         false);
     ADEBUG << "Skip path reference when changing lane.";
+    ADEBUG << "valid_path_reference_counter[" << valid_path_reference_counter_
+           << "] total_path_counter[" << total_path_counter_ << "]";
+    return Status::OK();
+  }
+
+  // skip using path reference during side pass
+  if (reference_line_info->is_path_lane_borrow()) {
```

Comment:
why skip?
is sidepass one of the major usages of learning path_reference?

Reply:
This is for static obstacles. We were talking about moving obstacles.

Reply:
NO. path_reference from learning is supposed to work more effectively (than plain lane follow) while SidePassing for both static and moving obstacles. 
We were talking about how to use it on SidePass for moving obstacles, in order to see "improvements" compared to rule-based planner, since rule-based planner currently is unable to support SidePass for moving obstacles. 
Unfortunately the conclusion was, there's too much work at this point because our current code doesn't support it at all. So we decided to work on NUDGE on moving obstacle first (no change on code level, but more on data/model side).

Reply:
I'll merge for now. 
Need updating later.